std/int128.d

```diff
@@ -0,0 +1,374 @@
+// Written in the D programming language
+/**
+ * Implements a signed 128 bit integer type.
+ *
+    Author:     Walter Bright
+    Copyright:  Copyright (c) 2022, D Language Foundation
+    License:    $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0)
+    Source:     $(PHOBOSSRC std/int128.d)
+ */
+module std.int128;
+
+private import core.int128;
+
+
+/***********************************
+ * 128 bit signed integer type.
+ */
+
+public struct Int128
+{
+  @safe pure nothrow @nogc:
+
+    Cent data;          /// core.int128.Cent
+
+    /****************
```

Comment:
Use the right style in documentation comments

https://dlang.org/dstyle.html#phobos_documentation

Reply:
I used the correct style. Dscanner does not work.

Reply:
This is not about Dscanner, and you're not using the correct style. There's extra stars.

> - Documentation comments should not use more than two stars /** or two pluses /++ in the header line.
> (...)
> - Documentation comments should not have leading stars on each line.

https://dlang.org/dstyle.html#phobos_documentation